setup.py

```diff
@@ -65,11 +65,10 @@
         url=URL,
         long_description=LONG_DESCRIPTION,
         long_description_content_type="text/x-rst",
-        packages=find_packages(),
+        packages=find_packages(exclude=["tests*"]),
         # because of an upload-size limit by PyPI, we're temporarily removing docs from the tarball.
         # Also see MANIFEST.in
         # package_data={'docs': ['*']},
-        include_package_data=True,
```

Comment:
Doesn't this exclude other stuff like example NBs or csv files?

Reply:
There is no more csv files in `pymc` (I think they moved to `pymc-examples`), and if NBs were ever included, I don't think they are not anymore. As far as I can say, installing `pymc` with or without this line produces the wheel.